.github/terraform/scripts/verify_member_count.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+#set -o pipefail
```

Comment:
do we need it? or why it is commented out?

Reply:
Answered on above comment.